data/remnant/remnant outfits.txt

```diff
@@ -51,6 +51,42 @@ effect "inhibitor impact"
 	"random velocity" 6
 	"velocity scale" -.05
 
+outfit "Inhibitor Turret"
+	category "Turrets"
+	licenses
+		Remnant
+	cost 1371000
+	thumbnail "outfit/inhibitor turret"
+	"mass" 32
+	"outfit space" -25
+	"weapon capacity" -25
+	"required crew" 1
```

Comment:
```suggestion
	"required crew" 3
```

Reply:
No, I do not think that a higher crew count is appropriate here. Higher price is definitely worth considering, but figuring out a way to miniaturize stuff so that more of it is in the external turret isn't something that would lend itself to having triple the crew count to utilize it.